resources/js/components/modals/concept/Create.vue

```diff
@@ -70,15 +70,18 @@
                             </div>
                         </div>
                         <input
+                            ref="inputField"
                             type="text"
                             class="form-control"
                             v-model="state.concept.label"
+                            @keydown.enter.prevent.stop="() => submitButton.click()"
```

Comment:
Is this really required? enter should already submit the form (same for clicking the submit button).
For me it also works without the `keydown` event handler.